### PR TITLE
Broken link fixes. 

### PR DESCRIPTION
**Pull request by TheMythoclast**
Broken link fixes, please let me know if there are any issues.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=26672) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=26672) before we can accept your contribution.<br/><br/>:x: TheMythoclast [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=26672)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=26672) it.</sub>

**Review by guardrex — DISMISSED** [review]
Hello @TheMythoclast ...

The MS doc links to `https://docs.microsoft.com/en-us` should be relative and without the localization segment. There are four of them on the build report to update.

**Comment by guardrex** [comment]
Sorry, Rick ... I didn't know you were just about to ping Wade. 😄

File: aspnetcore/data/ef-rp/migrations.md [hunk 1]
```diff
@@ -297,7 +297,7 @@ EF Core uses the `__MigrationsHistory` table to see if any migrations need to ru
 ## Troubleshooting
 
 Download the [completed app](
-https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/data/ef-rp/intro/samples/cu21snapshots/part-4-migrations).
+https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/data/ef-rp/intro/samples/cu21snapshots/Part-4-Migrations).
```
Comment (Rick-Anderson) [inline aspnetcore/data/ef-rp/migrations.md:300]: I'd revert this and the other case changes. Only misconfigured servers have case sensitive URL.

File: aspnetcore/migration/50-to-60-samples.md [hunk 2]
```diff
@@ -237,7 +237,7 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
 
 ## Custom dependency injection (DI) container
 
-The following .NET 5 and .NET 6 samples use [Autofac](https://autofac.readthedocs.io/latest/integration/aspnetcore.html)
+The following .NET 5 and .NET 6 samples use [Autofac](https://docs.autofac.org/en/latest/integration/aspnetcore.html)
```
Comment (Rick-Anderson) [inline aspnetcore/migration/50-to-60-samples.md:240]:
```suggestion
The following .NET 5 and .NET 6 samples use [Autofac](https://docs.autofac.org/latest/integration/aspnetcore.html)
```

File: aspnetcore/test/troubleshoot-azure-iis.md [hunk 3]
```diff
@@ -834,7 +834,7 @@ For more information, see <xref:host-and-deploy/iis/logging-and-diagnostics#enha
 When an app responds slowly or hangs on a request, see the following articles:
 
 * [Troubleshoot slow web app performance issues in Azure App Service](/azure/app-service/app-service-web-troubleshoot-performance-degradation)
-* [Use Crash Diagnoser Site Extension to Capture Dump for Intermittent Exception issues or performance issues on Azure Web App](https://blogs.msdn.microsoft.com/asiatech/2015/12/28/use-crash-diagnoser-site-extension-to-capture-dump-for-intermittent-exception-issues-or-performance-issues-on-azure-web-app/)
+* [Use Crash Diagnoser Site Extension to Capture Dump for Intermittent Exception issues or performance issues on Azure Web App](https://docs.microsoft.com/en-us/azure/app-service/troubleshoot-performance-degradation)
```
Comment (Rick-Anderson) [inline aspnetcore/test/troubleshoot-azure-iis.md:837]: `/en`

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Comment by Rick-Anderson** [comment]
image

File: aspnetcore/data/ef-rp/migrations.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/data/ef-rp/migrations.md:300]: I checked on the "SameSite" one ... *Indeed!* They ain't doing the right thing on that server. It 404's if lowercase.

File: aspnetcore/test/troubleshoot-azure-iis.md (hunk 3 above)
Comment (TheMythoclast) [reply aspnetcore/test/troubleshoot-azure-iis.md]: Fixed, sorry about that.

File: aspnetcore/test/troubleshoot-azure-iis.md [hunk 4]
```diff
@@ -1284,7 +1284,7 @@ For more information, see <xref:host-and-deploy/aspnet-core-module#log-creation-
 When an app responds slowly or hangs on a request, see the following articles:
 
 * [Troubleshoot slow web app performance issues in Azure App Service](/azure/app-service/app-service-web-troubleshoot-performance-degradation)
-* [Use Crash Diagnoser Site Extension to Capture Dump for Intermittent Exception issues or performance issues on Azure Web App](https://blogs.msdn.microsoft.com/asiatech/2015/12/28/use-crash-diagnoser-site-extension-to-capture-dump-for-intermittent-exception-issues-or-performance-issues-on-azure-web-app/)
+* [Use Crash Diagnoser Site Extension to Capture Dump for Intermittent Exception issues or performance issues on Azure Web App](https://docs.microsoft.com/azure/app-service/troubleshoot-performance-degradation/)
```
Comment (Rick-Anderson) [inline aspnetcore/test/troubleshoot-azure-iis.md:1287]:
```suggestion
* [Use Crash Diagnoser Site Extension to Capture Dump for Intermittent Exception issues or performance issues on Azure Web App](/azure/app-service/troubleshoot-performance-degradation/)
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@TheMythoclast thanks for the PR.

**Comment by wadepickett** [comment]
I was trying to get two other PR's in before hitting this one today.  Thanks everyone for jumping in just the same.

**Pull request merged**